--- ifhp-3.5.10/configure.in~	Thu Jun 27 01:47:23 2002
+++ ifhp-3.5.10/configure.in	Sun Jul 28 11:07:55 2002
@@ -17,34 +17,14 @@
 AC_DEFINE_UNQUOTED(VERSION, "$VERSION")
 AC_SUBST(PACKAGE)
 AC_SUBST(VERSION)
-ALL_LINGUAS="fr"
 
 AC_PREFIX_DEFAULT(/usr/local)
 AC_SUBST(INSTALL)
 AC_SUBST(INSTALL_MAN)
-AC_SUBST(SHELL)
-AC_SUBST(CPP)
-AC_SUBST(CFLAGS)
-AC_SUBST(CPPFLAGS)
-AC_SUBST(LDFLAGS)
-AC_SUBST(LDCC)
 
 dnl check for awk
 AC_PROG_AWK
-AC_SUBST(AWK)
-AC_PATH_PROG(PERL,perl)dnl
-AC_SUBST(PERL)dnl
-
-WITH_CCOPTS dnl
-WITH_CPPOPTS dnl
-WITH_LDOPTS dnl
-
-if test -z "$LOCALEDIR" ; then
-    LOCALEDIR='$(prefix)/share/locale'
-fi;
-
-WITH_LOCALEDIR dnl
-
+AC_PATH_PROG(PERL,perl)
 
 dnl --------------------------------------------------------------------------
 dnl ------- XXXXXX START OF OPTIONS ------------------------------------------
@@ -113,16 +93,6 @@
 AC_MSG_RESULT($FONT_DIR)
 AC_SUBST(FONT_DIR)
 
-dnl default strip binaries 
-AC_MSG_CHECKING(disable stripped binaries)
-AC_ARG_ENABLE( strip,
-[  --disable-strip          disable stripping binaries by default],
-[v=no; STRIP_OPTION=""; ],
-[v=yes;  STRIP_OPTION="-s"; ]
-)
-AC_MSG_RESULT($v)
-AC_SUBST(STRIP_OPTION)
-
 dnl dislabel ghostscript check
 AC_MSG_CHECKING(disable GhostScript check)
 AC_ARG_ENABLE( gscheck,
@@ -1127,43 +1097,11 @@
 
 AC_PROG_MAKE_SET
 
-dnl ----------------- force this to be Bourne Shell for now ---------------
-AC_MSG_CHECKING(for shell)
-SHELL=/bin/sh
-AC_MSG_RESULT(using $SHELL (FORCED))
-
 dnl ----------------- Option for enabling GNU gettext --------------------
 
+ALL_LINGUAS="fr"
 AM_GNU_GETTEXT
 
-AC_MSG_RESULT([LIBS $LIBS, INTLLIBS $INTLLIBS])
- if test "${USE_NLS}" = yes; then
-   AC_MSG_RESULT([Using NLS, localedir ${LOCALEDIR}, need intl/libintl.a - "${USE_INCLUDED_LIBINTL}" ])
-   if test "${USE_INCLUDED_LIBINTL}" = yes ; then
-      CPPFLAGS="$CPPFLAGS -I \${top_builddir}/intl"
-	  AC_MSG_RESULT([using \${top_builddir}/intl])
-	  if test "$nls_cv_force_use_gnu_gettext" = no ; then
-		echo "NLS enabled and requested system gettext support not available."
-		echo " You may need to use:"
-		echo "  configure '--with-cppopts=-I/usr/local/include' '--with-ldopts=-L/usr/local/lib'"
-		echo " See ${srcdir}/ABOUT-NLS.LPRng in the LPRng distribution for details."
-		exit 1;
-      fi
-	  if test ! -f ${srcdir}/intl/gettext.h ; then
-		echo "gettext code is not in the ${srcdir}/intl directory."
-		echo "See ${srcdir}/ABOUT-NLS.LPRng in the LPRng distribution for details."
-		exit 1;
-	  fi
-   fi
-   LIBS="$INTLLIBS $LIBS"
- else
-   AC_MSG_RESULT([No NLS])
- fi   
- if test "$XGETTEXT" = "" ; then
-	 XGETTEXT=xgettext;
- fi
- AC_MSG_RESULT(using XGETTEXT $XGETTEXT)
-
 dnl ----------------------------------------------------------------------------
 dnl ----------------- END OF GENERAL CONFIGURATION   ---------------------------
 
@@ -1171,19 +1109,14 @@
 dnl ----------------- START OF OUTPUT ------------------------------------------
 
 
-if test "$ac_cv_prog_gcc" = yes; then
-	CPPFLAGS="$CPPFLAGS -g -W -Wall -Werror -Wno-unused "
-	CFLAGS="$CFLAGS -g -W -Wall -Werror -Wno-unused "
-fi;
-
-AC_OUTPUT(
+AC_OUTPUT([
 Makefile src/Makefile man/Makefile fonts/Makefile
 HOWTO/Makefile
 UTILS/accounting.sh UTILS/sendhp.sh UTILS/supported
 UTILS/fixupdate
 src/wrapper
 po/Makefile.in intl/Makefile
-)
+])
 
 dnl ----------------------------------------------------------------------------
 dnl ----------------- END OF OUTPUT --------------------------------------------
--- ifhp-3.5.10/src/Makefile.in~	Sun Jul 28 11:08:34 2002
+++ ifhp-3.5.10/src/Makefile.in	Sun Jul 28 11:08:34 2002
@@ -12,15 +12,15 @@
 sysconfdir=@sysconfdir@
 libexecdir=@libexecdir@
 libdir=@libdir@
+datadir=@datadir@
 top_builddir=..
 
 INSTALLCMD=@INSTALL@
 FILTER_DIR=@FILTER_DIR@
 IFHP_CONF=@IFHP_CONF@
 CC      = @CC@
-LDCC      = @LDCC@
 LDFLAGS= @LDFLAGS@
-LOCALEDIR= @LOCALEDIR@
+LOCALEDIR= $(datadir)/locale
 # add additional definitions here
 
 # libraries you will need (found by config), add more to end
@@ -42,8 +42,7 @@
 	-I. -I.. -I${SRC} $(TEST_FLAGS) \
 	-DIFHP_CONF=\"$(IFHP_CONF)\" -DLOCALEDIR=\"$(LOCALEDIR)\"
 
-LINK =  $(LDCC) $(LDFLAGS) -o $@
-STRIP=@STRIP_OPTION@
+LINK =  $(CC) $(LDFLAGS) -o $@
 
 VPATH=.:..:${SRC}
 
@@ -90,7 +89,7 @@
 install: all
 	if [ ! -d ${DESTDIR}$(FILTER_DIR) ] ; then ${SRC}/mkinstalldirs ${DESTDIR}$(FILTER_DIR); fi;
 	for i in $(COMMANDS) ; do \
-		$(INSTALLCMD) ${STRIP} -m 755 $$i ${DESTDIR}$(FILTER_DIR); \
+		$(INSTALLCMD) -m 755 $$i ${DESTDIR}$(FILTER_DIR); \
 	done ;
 	for i in $(INSTALL) ; do \
 		$(INSTALLCMD) -m 755 $$i ${DESTDIR}$(FILTER_DIR)/`basename $$i`; \
--- ifhp-3.5.10/Makefile.in~	Sat Jul  6 22:10:28 2002
+++ ifhp-3.5.10/Makefile.in	Tue Jul 30 20:33:15 2002
@@ -48,10 +48,7 @@
 
 # we use this to do recursion to specified directories
 $(ALLDIRS):
-	if [ "$@" != po -o "$(USE_NLS)" != "no" ] ; then \
-		cd $@; $(MAKE) localedir=${DESTDIR}${LOCALEDIR} \
-		DESTDIR=$(DESTDIR) $(MAKETARGET) ; \
-	fi
+	cd $@; $(MAKE) DESTDIR=${DESTDIR}
 
 ###############################################################################
 
