--- ifhp-3.5.10/src/ifhp.c.orig	Mon Jul  8 17:58:41 2002
+++ ifhp-3.5.10/src/ifhp.c	Sun Jul 28 10:57:02 2002
@@ -2377,7 +2377,7 @@
 						if( value[0] == '[' ){
 							++value;
 							while(isspace(cval(value))) ++value;
-							if( t > value && cval(t-1) == ']' ) *t = 0;
+							if( t > value && cval(t-1) == ']' ) *(t-1) = 0;
 							for( t = value+safestrlen(value); t > value && isspace(cval(t-1)); --t );
 						}
 					}
